build/js/main.js

```diff
@@ -63,13 +58,55 @@ function anchorifyPage() {
         if (hasAttr($(this), 'id')) {
             var anchor = '#' + $(this).attr('id');
             $(this).append('<a href="' + window.location.origin + window.location.pathname + anchor + '" class="headingAnchor">' + anchorSVG + '</a>');
-            scrollToInit($(anchor), function(){
+            headingScrollInit($(anchor), function(){
```

Comment:
Instead of targeting the table of contents and the headers specifically, why not just target all anchors with hashes by using something like: `$("a[href*=#]")`. On some pages we have lists of links that should also use the smooth scroll. It would make the js simpler and more generic. It would also not be a bad idea to use the table of contents instead of the lists now too, but that should be a separate PR.